src/odemis/acq/millmng.py

```diff
@@ -0,0 +1,527 @@
+                raise
+            # Configure HoV
+            self._scanner.horizontalFoV.value = milling_settings.horizontalFoV.value
+
+            # Make the blanker automatic (ie, disabled when acquiring)
+            self._scanner.blanker.value = None
+
+            # set the milling current
+            self._restore_beam_shift(milling_settings.current.value)
+
+            # Initialize the drift corrector only if it is requested by the user
+            # max_pixels can change according to drift correction approach
+            if milling_settings.dcRoi.value == UNDEFINED_ROI:
+                pass
+            else:
+                # Change the current to the drift correction current and keep the beam shift value unchanged
+                self._restore_beam_shift(milling_settings.dcCurrent.value)
+                drift_est = AnchoredEstimator(self._scanner, self._detector,
+                                              milling_settings.dcRoi.value,
+                                              milling_settings.dcDwellTime.value,
+                                              max_pixels=ANCHOR_MAX_PIXELS, follow_drift=False)
+                # acquire an image at the given location (RoI)
+                drift_est.acquire()
+                da = drift_est.raw[-1]
+                pxs_anchor = da.metadata[model.MD_PIXEL_SIZE]
+                if self._log_path:
+                    fn = f"{self._fn_bs}-dc-{site.name.value}-{milling_settings.name.value}-0{self._fn_ext}"
+                    self._exporter.export(os.path.join(self._log_dir, fn), [da])
+
+            # Compute the probe size and overlap, based on the requested pixel size
+            # For the probe size, use the pixel size in the largest dimension (typically X)
+            # To adjust the pixel size in Y, compute the overlap so that probe_size * (1-overlap)) == pixelSize.
+            # X is horizontal axis, Y is vertical axis
+            pxs = milling_settings.pixelSize.value
+            probe_size = max(pxs)
+            overlap = [1 - (pxs[0] / probe_size), 1 - (pxs[1] / probe_size)]  # Always >= 0
+
+            if self._future._task_state == CANCELLED:
+                raise CancelledError()
+
+            hori_dim = (milling_settings.roi.value[2] - milling_settings.roi.value[0]) * \
+                       milling_settings.pixelSize.value[0]
+            vert_dim = (milling_settings.roi.value[3] - milling_settings.roi.value[1]) * \
+                       milling_settings.pixelSize.value[1]
+            dims = [hori_dim, vert_dim]
+
+            hori_t = ((milling_settings.roi.value[2] - milling_settings.roi.value[0]) / 2 - 0.5) * \
+                     milling_settings.pixelSize.value[0]
+            vert_t = ((milling_settings.roi.value[3] - milling_settings.roi.value[1]) / 2 - 0.5) * \
+                     milling_settings.pixelSize.value[1]
+            trans = [hori_t, vert_t]
```

Comment:
The correct formula should use the horizontalFoV instead of pixelSize. However, I'd suggest to drop all of this, and the logging.info(). `mill_rectangle()` is going to log almost the same thing. 
Possibly, you could move this logging to `do_fake_milling()` in the test cases, as that's a case where it would be handy.